ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/EeaSendTransaction.java

```diff
@@ -31,79 +32,87 @@
       "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675";
 
   public Request<Object, EthSendTransaction> withGas(final String gas) {
-    return createRequest(defaultTransaction().withGas(gas));
+    return createRequest(defaultTransaction().withGas(gas).build());
   }
 
   public Request<Object, EthSendTransaction> withGasPrice(final String gasPrice) {
-    return createRequest(defaultTransaction().withGasPrice(gasPrice));
+    return createRequest(defaultTransaction().withGasPrice(gasPrice).build());
   }
 
   public Request<Object, EthSendTransaction> withValue(final String value) {
-    return createRequest(defaultTransaction().withValue(value));
+    return createRequest(defaultTransaction().withValue(value).build());
   }
 
   public Request<Object, EthSendTransaction> withNonce(final String nonce) {
-    return createRequest(defaultTransaction().withNonce(nonce));
+    return createRequest(defaultTransaction().withNonce(nonce).build());
   }
 
   public Request<Object, EthSendTransaction> withSender(final String sender) {
-    return createRequest(defaultTransaction().withFrom(sender));
+    return createRequest(defaultTransaction().withFrom(sender).build());
   }
 
   public Request<Object, EthSendTransaction> withReceiver(final String sender) {
-    return createRequest(defaultTransaction().withTo(sender));
+    return createRequest(defaultTransaction().withTo(sender).build());
   }
 
   public Request<?, EthSendTransaction> withData(final String data) {
-    return createRequest(defaultTransaction().withData(data));
+    return createRequest(defaultTransaction().withData(data).build());
+  }
+
+  public Request<?, EthSendTransaction> withPrivateFor(final List<String> privateFor) {
+    return createRequest(defaultTransaction().withPrivateFor(privateFor).build());
+  }
+
+  public Request<?, EthSendTransaction> withRestriction(final String restriction) {
+    return createRequest(defaultTransaction().withRestriction(restriction).build());
   }
 
   public Request<?, EthSendTransaction> missingSender() {
-    return createRequest(defaultTransaction().removeFrom());
+    return createRequest(transactionWithoutField("from"));
```

Comment:
is it worth creating String constants for "from" , "nonce", "to" etc. ?

Reply:
I think it is, I meant to do that actually but forgot.
done